### DISCUSSION TITLE
Multiple locks?

### DISCUSSION THREAD

**Discussion by joeymc93** [technyon/nuki_hub#480] [Q&A]
Have successfully paired and am controlling with Home Assistant, can my Olimex board pair to multiple Nuki locks?

**Comment by mundschenk-at** [comment]
No.

**Comment by technyon** [comment]
Hi,

The longer answer is we thought about it, but decided against it. Multiple locks would add a significant complexity to the code, which needs to be maintained and opens the door for new bugs. Considering this is not a very common scenario since you want the ESP to be close to the door, we decided against it. Plus, ESPs are cheap enough that it doesn't add a lot of cost.

**Reply by joeymc93** [reply]
Hey mate thanks for the response, will pick up another board
